frontend/src/utilities/calendar.ts

```diff
@@ -100,14 +100,26 @@ function datesInBreak(
     const dates: string[] = [];
     for (
       const date = start;
-      date.getTime() < end;
+      date.getTime() <= end;
       date.setUTCDate(date.getUTCDate() + 1)
     )
       if (days.includes(date.getUTCDay())) dates.push(isoString(date, time));
     return dates;
   });
 }
 
+function transferDays(
+  transfers: SeasonCalendar['transfers'],
+  days: number[],
+  time: string,
+) {
+  return transfers.flatMap((t) => {
```

Comment:
You are only returning strings. Shouldn't this be `map` instead of `flatMap`?

Reply:
right, either works in this case but `map` makes more sense.